@xen-orchestra/web/src/pages/backup/[id]/backupTargets.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <UiCard class="backupTarget">
+    <StorageRepositoriesTable :storage-repositories-targets />
+    <BackupRepositoriesTable :backup-repositories-targets />
```

Comment:
```suggestion
    <StorageRepositoriesTargetsTable :storage-repositories-targets />
    <BackupRepositoriesTargetsTable :backup-repositories-targets />
```
I wonder if we shouldn't name the component that way instead. 🤔